src/browser/screens/Fifth.react.js

```diff
@@ -115,10 +106,9 @@ export default class FifthScreen extends Component {
     }
 
     this.cvFile = files[0];
-    fileMimeType = this.cvFile.type;
-    fileSize = this.bytesToMegaBytes(this.cvFile.size);
-
-    validMimeType = VALID_MIME_TYPES.some((mimeType) => {
+    let fileMimeType = this.cvFile.type;
+    let fileSize = this.cvFile.size / BYTE_TO_MEGABYTE_RATIO;
+    let validMimeType = VALID_MIME_TYPES.some((mimeType) => {
```

Comment:
shouldn't those `lets` be a `consts` instead? Are those being mutated along the way?